app/services/check_answers/applicant_section.rb

```diff
@@ -30,11 +30,7 @@ def from_session(session)
     end
 
     def display_fields
-      if partner
-        FIELDS
-      else
-        FIELDS - %i[partner_employed partner_over_60]
```

Comment:
You should be able to delete the partner_employed and partner_over_60 attributes on lines 23 and 24 here, because they're no longer things that appear in the applicant section of check answers